src/Tribe/App_Shop.php

```diff
@@ -77,11 +79,25 @@ public function add_toolbar_item() {
 		}
 
 		/**
+		 * Registers the plugin assets
+		 */
+		protected function register_assets() {
+			tribe_assets(
+				Tribe__Main::instance(),
+				array(
+					array( 'app-shop-css', 'app-shop.css' ),
+					array( 'app-shop-js', 'app-shop.js' ),
+				),
+				'admin_print_styles-' . $this->admin_page
```

Comment:
I _think_ if we switch this out for `admin_enqueue_scripts` we'll be laughing all the way to the bank and can strip out the calls in the `enqueue()` method/remove that method entirely